std/base64.d

```diff
@@ -2094,6 +2095,8 @@ class Base64Exception : Exception
     // Verify that any existing workaround that uses & still works.
     InputRange ir2;
     OutputRange or2;
-    assert(Base64.decode(ir2, &or2) == 6);
+    () @trusted {
```

Comment:
This seems kind of besides the point. decode is the function really being tested here, so it makes sense to leave this unit test system if it's system. Same for the above test

Reply:
Did you look the the full context?
This is just a legacy line - the real test is five lines above.

```d
assert(Base64.decode(ir, or) == 6);
assert(or.result == [0x1a, 0x2b, 0x3c, 0x4d, 0x5d, 0x6e]);
 	 
// Verify that any existing workaround that uses & still works
```